### PR TITLE
[CDAP-586] add a test case for querying non-partitioned file sets

### PR DESCRIPTION
**Pull request by anew**

### CODE REVIEW

File: cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.apache.avro.generic.GenericDatumWriter;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.io.DatumWriter;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * A helper to deal with Avro schema, records, serializations, etc.
+ * This is in its own class because it uses Avro the Schema class, which would clash with our own Schema elsewhere.
```
Comment (awholegunch) [inline cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java]: Avro as the Schema class

File: cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ */
+public class AvroHelper {
+
+  /**
+   * Generate an Avro file of schema (key: String, value String) containing record of the form ("i", "Record #i)
```
Comment (awholegunch) [inline cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java]: (key: String, value: String) containing records of

Comment (awholegunch) [inline cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java]: ("i", "Record #i")

**Comment by anew** [comment]
Build is running here: https://builds.cask.co/browse/CDAP-RBT129

**Comment by awholegunch** [comment]
A few minor doc suggestions...

File: cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java (hunk 1 above)
Comment (anew) [inline cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java]: Avro is not a Schema class. It is a fie format, and its schema class is called "Schema"

Comment (awholegunch) [inline cdap-explore/src/test/java/co/cask/cdap/explore/service/AvroHelper.java]: Avro's Schema class \[?]

**Comment by anew** [comment]
Build succeeded. Waiting for thumbs up.

**Comment by albertshau** [comment]
:+1:

**Pull request merged**